examples/dev-mashup/package.json

```diff
@@ -8,13 +8,14 @@
     "build": "parcel build index.html"
   },
   "dependencies": {
-    "@nebula.js/stardust": "latest",
     "@nebula.js/sn-bar-chart": "latest",
     "@nebula.js/sn-line-chart": "latest",
+    "@nebula.js/stardust": "latest",
     "enigma.js": "2.7.2"
   },
   "devDependencies": {
     "@babel/core": "7.12.3",
-    "parcel": "^2.0.0-rc.0"
+    "parcel": "^2.2.1",
+    "parcel-bundler": "^1.12.5"
```

Comment:
I had to add the parcel-bundler to get the `dev-mashup` to run.

Reply:
it feel very wrong to use both 1.x and 2.x version of parcel.
why was it needed?
what whent wrong?

Reply:
Try this:
```
cd examples/dev-mashup
yarn && yarn start
```

I get this error:
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/5780544/150944601-32cd0e58-6f2d-42ba-bd0f-a5a755e37e4a.png">

It works when including the `parcel-bundler`.

Reply:
Unless you already have a better workaround, I suggest we keep this until finding a better solution. Right now it's broken.

Reply:
following the upgrade guide for Parcel 2
https://parceljs.org/getting-started/migration
(and remove parcel-bundler)

or

remove  `parcel": "^2.2.1`

I get cryptic `error Command failed with exit code 3221226356.` using parcel 2 on windows most of the time (but not allways) so using `parcel-bundler` may be safer (but deprecated) but then you should only have that dependency

Reply:
I removed `parcel`, only using `parcel-bundler` now and works on my machine (Mac).